content/hardware/04.pro/carriers/portenta-max-carrier/tutorials/x8-getting-started/content.md

```diff
@@ -0,0 +1,216 @@
+---
+title: 'Getting Started with Portenta Max Carrier and Portenta X8'
+difficulty: easy
+description: "Get started with your Portenta Max Carrier and Portenta X8"
+tags:
+  - Installation
+  - Getting started
+author: 'Benjamin Dannegård, José Bagur, Taddy Chung'
+hardware:
+  - hardware/04.pro/carriers/portenta-max-carrier
+  - hardware/04.pro/boards/portenta-x8
+software:
+  - ide-v1
+  - ide-v2
+  - web-editor
+---
+
+## Introduction 
+
+Using the Portenta X8 with the Portenta Max Carrier enables a large suite of peripherals and features. With the Portenta X8 it is possible to use Linux containers and python scripts, it is also possible to use the M4 core on the board to interface with the Portenta Max Carrier. In this tutorial we will focus on the peripherals and features that are available to us.
+
+## Goals
+
+The goals of this project are:
+
+- Learn how to connect the Portenta X8 and Portenta Max Carrier
+- Learn about the Portenta Max Carrier's headers, peripherals and connectors
+
+### Required Hardware and Software
+
+- [Arduino® Portenta X8](https://store.arduino.cc/products/portenta-x8)
+- [Arduino® Portenta Max Carrier](https://store.arduino.cc/products/portenta-max-carrier)
+
+## The Portenta Max Carrier
+
+The Portenta Max Carrier was designed to add capabilities to the Arduino® Portenta X8 board and provide easy access to its on-board peripherals. It is designed to enable research and develop industrial grade advanced applications, from fast prototyping to deployable robust single board computer.
```

Comment:
We can remove the "Arduino" after mentioning it the first time (add this in the introduction and remove from the rest) ⭐